### PR TITLE
Handle parenthesised future imports

### PR DESCRIPTION
**Pull request by takluyver**
Modernising code that did `from __future__ import (...)` was failing.

I couldn't get the test for this to fail - I'm not sure why. But this change made it work on the actual code I was testing with.

### CODE REVIEW

**Comment by takluyver** [comment]
I figured out how to tweak the test so that it does fail without this change; though I'm still not really sure why my first attempt at the test wasn't failing.

File: tests/test_future_behaviour.py [hunk 1]
```diff
@@ -111,8 +111,21 @@ def test_problematic_file():
...
+)
+
 def test_future_import_as():
     check_on_input(*FUTURE_IMPORT_AS)
 
 def test_future_import_as_multiple():
     check_on_input(*FUTURE_IMPORT_AS_MULTIPLE)
+
+def test_future_import_paren():
+    check_on_input(*FUTURE_IMPORT_PAREN)
```
Comment (brettcannon) [inline tests/test_future_behaviour.py]: Don't know if it matters, but there is a missing newline at the end of the file.

**Comment by brettcannon** [comment]
LGTM except for a missing newline at the end of the test file.

File: tests/test_future_behaviour.py (hunk 1 above)
Comment (takluyver) [inline tests/test_future_behaviour.py]: I don't think it really matters, but I've added it back to make things tidier.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**